src/components/SegmentedProgressBar/SegmentedProgressBar.unit.spec.jsx

```diff
@@ -14,5 +15,42 @@ describe('<SegmentedProgressBar/>', () => {
   });
 
   it('should pass aXe check', () =>
-    axeCheck(<SegmentedProgressBar current={2} total={5} label="aria label here"/>));
+    axeCheck(
+      <SegmentedProgressBar current={2} total={5} label="aria label here" />,
+    ));
+
+  describe('analytics event', function () {
+    it('should be triggered when a SegmentedProgressBar is updated', () => {
```

Comment:
Does the event trigger on initial mount as well, i.e. when `current` is 0 in this case? If so, can that be included in the tests?

Reply:
Yes, indeed.  An oversight on my part.  Updated!